roles/db-create/tasks/rac-db-create.yml

```diff
@@ -91,7 +91,7 @@
 
 - name: rac-db-create | Create DBCA response file
   become: yes
-  become_user: "{{ oracle_user }}"
+  become_user: root
```

Comment:
Is this change to address any error?
I do not recalling facing any issues with creating rsp file as OS user `oracle`.

Reply:
The (albeit rare) issue is that `oracle_user` is a parameter and could potentially change between runs, such as changes to privilege separation.  Crating the file as root and chowning it allows any existing files to be clobbered.